modules/core/02-client/keeper/events_test.go

```diff
@@ -0,0 +1,88 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	commitmenttypes "github.com/cosmos/ibc-go/v8/modules/core/23-commitment/types"
+	ibctm "github.com/cosmos/ibc-go/v8/modules/light-clients/07-tendermint"
+	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+)
+
+func (suite *KeeperTestSuite) TestMsgCreateClientEvents() {
+	suite.SetupTest()
+	path := ibctesting.NewPath(suite.chainA, suite.chainB)
+
+	path.EndpointA.Counterparty.Chain.NextBlock()
```

Comment:
this line might not be necessary anymore

Reply:
looks like the test fails without this.